ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -132,8 +133,8 @@ public class MySqlNamedBlobDb implements NamedBlobDb {
    * the server side
    */
   private static final String INSERT_QUERY =
-      String.format("INSERT INTO %1$s (%2$s, %3$s, %4$s, %5$s, %6$s, %7$s, %8$s, %9$s) VALUES (?, ?, ?, ?, ?, ?, ?, ?)",
-          NAMED_BLOBS_V2, ACCOUNT_ID, CONTAINER_ID, BLOB_NAME, BLOB_ID, DELETED_TS, VERSION, BLOB_STATE, BLOB_SIZE);
+      String.format("INSERT INTO %1$s (%2$s, %3$s, %4$s, %5$s, %6$s, %7$s, %8$s, %9$s, %10$s) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)",
```

Comment:
can you document steps for rolling out change? what if we have to rollback to older code after we added the column? 

should we consider rolling out schema changes first and then follow up with actual functional logic change?